CHANGELOG.md

```diff
@@ -4,6 +4,73 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v5.0.0](https://github.com/buildkite/elastic-ci-stack-for-aws/compare/v4.5.0...master) (DRAFT)
+
+### Added
+<!-- 
+    TODO: document the new lambda scaler properly as a new default and what it means. For now https://forum.buildkite.community/t/experimental-lambda-based-scaler/425 is a good starting point.
+-->
+* **Our blazing fast lambda scaler is now the default** which makes for much faster scaling in response to pending jobs [#575](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/575) (@lox)
+* **Windows support** on a new Windows Server 2019 based image [#546](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/546) ([jeremiahsnapp](https://github.com/jeremiahsnapp))
+* Support for [buildkite/image-builder](https://github.com/buildkite/image-builder) which can enable you to customize AMIs based off the ones we ship [#692](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/692) ([keithduncan](https://github.com/keithduncan))
+* Support for multiple security groups on instances [#667](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/667) ([jdub](https://github.com/jdub))
+* AMI and Lambda Scaler support more regions [#718](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/718) ([JuanitoFatas](https://github.com/JuanitoFatas))
+* Support for loading BuildkiteAgentTokenPath from Parameter Store [#601](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/601) ([jradtilbrook](https://github.com/jradtilbrook)), [#625](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/625) ([jradtilbrook](https://github.com/jradtilbrook))
+* Dev workflow and documentation tweaks to make it easier to work on the stack [#706](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/706), [#714](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/714) ([yob](https://github.com/yob)), [#664](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/664) ([KevinGrandon](https://github.com/KevinGrandon)), [#650](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/650) ([pda](https://github.com/pda))
```

Comment:
I think I lean towards dropping dev-only items from the changelog. They're super useful changes, but this is a **long** list and the easier it is for *users* of the stack to skim for things they care about, the better.

Reply:
Given it's just rendering out PR (name) it's still skimmable if you're viewing the rendered markdown, but I'm inclined to agree, I'll cut this one